src/main/java/io/appform/jsonrules/expressions/equality/EqualsExpression.java

```diff
@@ -42,8 +42,8 @@ public EqualsExpression() {
     }
 
     @Builder
-    public EqualsExpression(String path, Object value, PreOperation<?> preoperation) {
-        super(ExpressionType.equals, path, preoperation);
+    public EqualsExpression(String path, Object value, Boolean defaultResult, PreOperation<?> preoperation) {
```

Comment:
Don't use boxed types when not absolutely necessary. I will not repeat this command below. Fix this in all other places.